config/bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+use Cake\Core\Configure;
+
+Configure::write('Acl.classname', 'DbAcl');
+Configure::write('Acl.database', 'default');
```

Comment:
I would wrap these 2 values around `if (!Configure::check('Acl'))` So people can still load their configuration files before loading the plugin